test/unit/sdam/srv_polling.test.js

```diff
@@ -425,4 +425,35 @@ describe('Mongos SRV Polling', function () {
       });
     });
   });
+
+  it('should emit a topologyDescriptionChanged event upon receiving an SRV update', function () {
```

Comment:
we talked about refactoring the test setup into a describe block to isolate the test condition better in the `it`

Reply:
resolving as done